en/getting-started/first-game/index.md

```diff
@@ -370,173 +832,203 @@ export class GameManager extends Component {
+You can change the **roadLength** property of the **Inspector** panel to control the length of the runway.
 
-When previewing, the road is now automatically generated, however, because the __Camera__ does not follow the `Player`, the road behind cannot be seen. Changing the __Camera__ in the __Scene__ to be a child node of the `Player` can help solve this.
+Now, click the **Preview** button to see the automatically created runway in the open browser.
+
+## Follow the Camera
+
+We can notice that the Camera does not follow the Player node, hence we can not see the runway behind it. We can avoid this situation by making the camera a child node of Player.
 
 ![drag camera to player](./images/drag-camera-to-player.gif)
 
-Now, the __Camera__ will follow the Player's movement.
+Thus the Camera will follow the Player's movement, now by clicking the **Preview** button, we can watch the whole runway.
+
+## Add a Start Menu
 
-## Adding a start menu
+The User Interface(UI) is an essential part of a game, developers can inform users of some key information such as data, status, etc so that the user can have their strategy. We could also add the game name, an introduction, the production staff, and other information.
 
-The __start menu__ is an indispensable part of most any game. Add the game name, game introduction, production staff and other information here. Creating a simple start menu starts with some basic steps:
+Please refer to relevant documents such as [2D/UI](. /. /2d-object/index.md) to help understand the mechanics of the UI as a 2D part.
 
-1. Add a button called `Play`
+Then we will describe a UI workflow taking the Start Menu as an example.
+
+1. Create a new Button node in the **Hierarchy** panel and rename it PlayButton
 
     ![create button](./images/create-button.gif)
 
-    This operation creates a `Canvas` node, a `PlayButton` node, and a `Label` node. Because the UI component needs to be displayed under the parent node with `Canvas`, the editor will automatically add one when it finds that there is not a node with this component in the current __Scene__. After creating the button, change the `String` property of `cc.Label` on the `Label` node from `Button` to `Play.`
+    It can be noticed that the engine automatically creates a Canvas node in the **Hierarchy** panel, a Player button node, and a Label node. That is because any node that contains a UI component only be seen when it is under the `Canvas` node, so the editor will create a Canvas in case there is none.
+
+    Change the **String** property of the `cc.Label` component in the Label Node from Button to Play.
 
-2. Create an empty node named `StartMenu` under `Canvas` and drag `PlayButton` under it. We can switch to the 2D editing view for UI editing operations by clicking the 2D/3D button on the toolbar.
+    > You may notice that the version of Cocos Creator you are using automatically adds a new Camera node when creating a UI node
```

Comment:
You may notice that the Cocos Creator automatically adds a new "Camera" node under "Canvas" when you create a UI node